x-pack/dockerlogbeat/magefile.go

```diff
@@ -119,13 +118,13 @@ func createContainer(ctx context.Context, cli *client.Client, arch string) error
 	// build, wait for output
 	buildResp, err := cli.ImageBuild(ctx, buildContext, buildOpts)
 	if err != nil {
-		return errors.Wrap(err, "error building final container image")
+		return fmt.Errorf("error building final container image"+": %w", err)
 	}
 	defer buildResp.Body.Close()
 	// This blocks until the build operation completes
 	buildStr, errBufRead := ioutil.ReadAll(buildResp.Body)
 	if errBufRead != nil {
-		return errors.Wrap(err, "error reading from docker output")
+		return fmt.Errorf("error reading from docker output"+": %w", errBufRead)
```

Comment:
@cmacknz note there was a bug originally here... logged `err` instead of `errBufRead`